app/main.py

```diff
@@ -1,8 +1,58 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        if not isinstance(comfort_class and clean_mark, int):
+            raise TypeError(" Comfort class and clean mark"
+                            " must be only positive integer")
+        if not isinstance(brand, str):
+            raise TypeError("Brand must be only string")
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+    def __str__(self) -> str:
+        return f"Comfort class - {self.comfort_class}, " \
+               f"clean mark - {self.clean_mark}, brand - {self.brand}"
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        if not isinstance(clean_power and count_of_ratings, int):
```

Comment:
It is not needed here, You'll learn proper validation approaches a bit later.
Remove everywhere